recipes/llvm-core/all/conandata.yml

```diff
@@ -1,4 +1,14 @@
 sources:
+  "19.1.7":
+    "llvm":
+      url: https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.7/llvm-19.1.7.src.tar.xz
+      sha256: 96f833c6ad99a3e8e1d9aca5f439b8fd2c7efdcf83b664e0af1c0712c5315910
+    "cmake":
+      url: https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.7/cmake-19.1.7.src.tar.xz
+      sha256: 11c5a28f90053b0c43d0dec3d0ad579347fc277199c005206b963c19aae514e3
+    "third-party":
+      url: https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.7/third-party-19.1.7.src.tar.xz
+      sha256: b96deca1d3097c7ffd4ff2bb904a50bdd56bec7ed1413ffb0d1d01af87b72c12
```

Comment:
```suggestion
```

What is the case needed third-party? It only contains unittest and benchmark folders. Both are unrelated when consuming or packaging the project in most cases. The benchmark headers can be disabled by using the CMake definitions LLVM_INCLUDE_BENCHMARKS=False

Reply:
Removed.